WordPressLoginFlow/build.gradle

```diff
@@ -59,17 +66,9 @@ dependencies {
 
     api 'com.google.android.gms:play-services-auth:18.1.0'
 
-    // Share FluxC version from host project if defined, otherwise fallback to default
-    if (project.hasProperty("fluxCVersion")) {
-        implementation("com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:$fluxCVersion") {
-            exclude group: "com.android.support"
-            exclude group: "org.wordpress", module: "utils"
-        }
-    } else {
-        implementation("com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:1.17.0") {
```

Comment:
I'm still not fluent enough in how those stings used for identifying a package / library work… how come that changed here from `com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc` to just `org.wordpress:flux:` below instead?

_(I probably need to deep dive in the docs about those "identifiers" at some point to understand how they are defined in the lib being referenced etc and how Gradle resolves them…)_

Reply:
This change is a result of _where_ the artifact is published. You can think of these `.`s and `:`s as folder separator. So, `org.wordpress:fluxc` would mean that, in one of the repositories, there is an `org` folder, which contains `wordpress` folder which contains `fluxc` folder which contains the artifacts of different versions and the metadata information.

When we [switched from Jitpack to S3](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2083), we started using a different group (folder) for where the artifacts are published. Specifically, [these lines](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2083/files#diff-6f7ff885ce362d8ceca855583e0ac547f430562813a1feb5070dcbe357bb7aa5R112-R113) tell where the artifacts should be published for fluxc module.

[Jitpack](https://jitpack.io/docs/) suggests using `com.github.User:Repo` as a convention, so we are following that for libraries that we publish to Jitpack. However, typical maven convention for the group is the reverse domain which is `org.wordpress`, `org.wordpress-mobile` and `com.automattic` for us depending on the library.

Hope that clears it up for you!

Reply:
🤯  that definitively clears this up, thanks! (I'm starting to create a bunch of notes I'm saving with all your gradle/Android explanations 😛 😍 )

Reply:
@AliSoftware Happy to help!